client/src/components/Dashboard/Dashboard.styles.tsx

```diff
@@ -24,8 +26,50 @@ export const DashboardCurrent = styled.div<StylesProps>`
   }
 `;
 
+export const DashboardToolbar = styled.div<StylesProps>`
+  margin-bottom: 1rem;
+`;
+
 export const DashboardForecast = styled.div<StylesProps>`
   grid-area: forecast;
+  height: 16.775rem;
 `;
 
-export const DashboardToolbar = styled.div<StylesProps>``;
+export const DashboardForecastTypes = styled.ul<StylesProps>`
+  width: 33%;
+`;
+
+interface DashboardForecastTypeStyles extends StylesProps {
+  active: boolean;
+}
+
+export const DashboardForecastType = styled.li<DashboardForecastTypeStyles>`
+  font-size: 1.17rem;
+  font-weight: 700;
+  color: ${props => {
+    if (!props.active && props.theme === 'light') {
+      return theme.palette.black;
+    }
+    if (!props.active && props.theme !== 'light') {
+      return theme.palette.white;
+    }
+    if (props.active) {
+      return theme.palette.primary.dark;
+    }
```

Comment:
I try to follow eslint rules. In this case, https://eslint.org/docs/latest/rules/no-nested-ternary:
if (foo) {
  thing = bar;
} else if (baz === qux) {
  thing = quxx;
} else {
  thing = foobar;
}
But if you remember from before, I also get warning "Unnecessary 'else' after 'return'.eslint[no-else-return](https://eslint.org/docs/rules/no-else-return)".

Also Igor Korshuk said not to use ternary expressions, especially nested, since it makes code implicit, difficult to read.

Reply:
ok, looks like we have different opinions on that, you can follow eslint :)